calour/dsfdr.py

```diff
@@ -292,11 +291,13 @@ def dsfdr(data, labels, transform_type='rankdata', method='meandiff',
         # call the user-defined function of statistical test
         t = method(data, labels)
         tstat = t.copy()
+        t = np.abs(tstat)
```

Comment:
why need to be absolute value? could you add comment to explain?

Reply:
We need abs() since dsFDR performs a two-sided test. added comment in file.